packages/blockchain-api/deploy.sh

```diff
@@ -7,18 +7,17 @@ set -euo pipefail
 # -n: Name of the network, maps to App Engine 'service' (integration, production, etc.)
 
 NETWORK=""
-PROJECT=""
 
 while getopts 'n:p:' flag; do
   case "${flag}" in
     n) NETWORK="$OPTARG" ;;
-    p) PROJECT="$OPTARG" ;;
     *) echo "Unexpected option ${flag}" ;;
   esac
 done
 
 [ -z "$NETWORK" ] && echo "Need to set the NETWORK via the -n flag" && exit 1;
-[ -z "$PROJECT" ] && echo "Need to set the PROJECT via the -p flag" && exit 1;
+
+PROJECT="celo-mobile-${NETWORK}"
```

Comment:
Is it possible to throw an error if the project `celo-mobile-alfajoresstaging` for example is not yet set up?

Reply:
Good idea! I'll add a check for supported projects!

Reply:
Backtracking on this, `gcloud` naturally complains if the project doesn't exist. This is probably enough for now.

Example:
```
❯ ./deploy.sh -n pilot
pilot is not yet supported, try one of these: alfajores mainnet
Starting notification service deployment.
Copying over config file
Deploying to gcloud
ERROR: (gcloud.app.deploy) Permissions error fetching application [apps/celo-mobile-pilot]. Please make sure you are using the correct project ID and that you have permission to view applications on the project.
```